ClearBlade.js

```diff
@@ -1195,7 +1195,11 @@ if (!window.console) {
         URI: this.URI
       };
       var callCallback = function (err, data) {
-        _createItemList(err, data.DATA, options, callback);
+        if(err) {
+          _createItemList(err, data, options, callback);
```

Comment:
Do we need to call _createItemList if there's an error? Seems like we should shoot out to the callback